component/class/defaults.yml

```diff
@@ -65,8 +69,49 @@ parameters:
 
     appuioManaged: true
     tenantID: ${cluster:tenant}
+    salesOrderID: TBD
     quotasEnabled: false
     grpcEndpoint: host.docker.internal:9443
+
+    billing:
+      namespace: ${appcat:namespace}
+      clusterId: ${cluster:name}
```

Comment:
The `tenantID` is outside of `billing` and the `clusterId` inside. This is inconsistent (also once the ID is all capital and once camel case.

Reply:
We probably won't need this one now, anyway.